src/main/java/com/owncloud/android/lib/common/OwnCloudClientManager.java

```diff
@@ -193,28 +195,36 @@ public NextcloudClient getNextcloudClientFor(OwnCloudAccount account, Context co
         }
 
         if (client == null) {
-            // no client to reuse - create a new one
-            // TODO remove dependency on OwnCloudClientFactory
-            client = OwnCloudClientFactory.createNextcloudClient(account.getBaseUri(), context.getApplicationContext(),
-                    true);
+
+
             // TODO v2
             //client.getParams().setCookiePolicy(CookiePolicy.BROWSER_COMPATIBILITY);
             // enable cookie tracking
 
             //AccountUtils.restoreCookies(accountName, client, context);
 
             account.loadCredentials(context);
-            client.credentials = account.getCredentials().toOkHttpCredentials();
+            val credentials = account.getCredentials().toOkHttpCredentials();
```

Comment:
I'm fine adding `val` via lombok in java code but we should then decide to do so. What do you think @tobiasKaminsky @ezaquari? Just asking since this adds a kotlin language feature to java code. Not sure if we want to mix language concepts. I am not having a strong opinion on this. We should then just document such a rule.

Reply:
No, this was just a mistake, as I thought I am writing Kotlin code.
I will change this.